pandora-client-web/src/graphics/def.ts

```diff
@@ -24,19 +31,27 @@ export const COMPUTED_LAYER_ORDERING = [
 	'BELOW_BACK_HAIR',
 	'BACK_HAIR',
 
-	'BELOW_ARMS_BACK',
-	'ARMS_BACK',
-	'ABOVE_ARMS_BACK',
+	'BELOW_ARM_LEFT_BACK',
+	'ARM_LEFT_BACK',
+	'ABOVE_ARM_LEFT_BACK',
+
+	'BELOW_ARM_RIGHT_BACK',
+	'ARM_RIGHT_BACK',
+	'ABOVE_ARM_RIGHT_BACK',
```

Comment:
not sure how simple it would be but it might look more readable if this is generated by a function, same for the other sections that got duplicated

Reply:
I prefer intentionally leaving this more explicit, as it still is short and easy to validate by TS this way. It also to me does feel easier to read honestly, than introducing functions.